lang/mrustc/Portfile

```diff
@@ -2,14 +2,16 @@
 
 PortSystem      1.0
 PortGroup       github 1.0
+PortGroup       openssl 1.0
+PortGroup       compiler_blacklist_versions 1.0
 
-github.setup    thepowersgang mrustc 4bee8893e8f471d315c1d2d9535216b116cdfe23
+github.setup    thepowersgang mrustc 3ec8cd8b1a3431124d278baefc8d0e34dc69b930
 
-version         20220111
+version         20220115
```

Comment:
Doesn't really matter but the title says `20220125` (which I think is the correct value)

Reply:
Indeed!